docs/advanced-features/http-api/index.md

```
@@ -9,10 +9,10 @@ Watchtower has an [optional](../../configuration/arguments/index.md#http_api_mod
 
 ## Endpoints
 
-|              **Name**              | **Method** | **Endpoint**  |          **Parameters**           |                           **Description**                            |
-|:----------------------------------:|:----------:|:-------------:|:---------------------------------:|:--------------------------------------------------------------------:|
-|     [Update](#http_api_update)     |   `POST`   | `/v1/update`  | [`image`](#image_parameter_usage) | Triggers container updates and returns JSON results of the operation |
-| [Metrics](../metrics-api/index.md) |   `GET`    | `/v1/metrics` |                                   |  Exposes Prometheus-compatible metrics for monitoring and alerting   |
+|              **Name**              | **Method** | **Endpoint**  |                           **Parameters**                            |                           **Description**                            |
+|:----------------------------------:|:----------:|:-------------:|:-------------------------------------------------------------------:|:--------------------------------------------------------------------:|
+|     [Update](#http_api_update)     |   `POST`   | `/v1/update`  | [`image`](#image_parameter_usage), [`async`](#asynchronous_updates) | Triggers container updates and returns JSON results of the operation |
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)